packages/components/src/components/teaser/teaser.stories.ts

```diff
@@ -35,711 +43,149 @@ export default {
- */
-
-export const InsetAndOrientation = {
-  name: 'Inset x Orientation',
-  parameters: { controls: { exclude: ['inset', 'breakpoint', 'variant'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: {
-          type: 'attribute',
-          name: 'breakpoint',
-          values: [
-            {
-              value: '0',
-              title: 'breakpoint = 0'
-            },
-            {
-              value: '9999',
-              title: 'breakpoint = 9999'
-            }
-          ]
-        },
-        y: { type: 'attribute', name: 'inset' }
-      },
-      args,
-      constants: [
-        {
-          type: 'template',
-          name: 'style',
-          value: '<div style="margin-bottom: 40px;">%TEMPLATE%</div>'
-        },
-        {
-          type: 'attribute',
-          name: 'variant',
-          value: 'neutral-100'
-        }
-      ]
-    });
-  }
+  render: (args: any) => generateTemplate({ args })
 };
 
 /**
- * Teaser when the `meta` slot is empty in all possible combinations of `inset` and `orientation`.
+ * The header and main slot are both obligatory.
+ * - Headers can be used to display titles and should always contain a `<h*>` element.
+ * - The default slot can display any content.
  */
 
-export const NoMeta = {
-  name: 'Empty Meta Slot',
-  parameters: { controls: { exclude: ['meta', 'breakpoint', 'inset', 'variant'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: {
-          type: 'attribute',
-          name: 'breakpoint',
-          values: [
-            {
-              value: '0',
-              title: 'breakpoint = 0'
-            },
-            {
-              value: '9999',
-              title: 'breakpoint = 9999'
-            }
-          ]
-        },
-        y: { type: 'attribute', name: 'inset' }
-      },
-      args,
-      constants: [
-        {
-          type: 'template',
-          name: 'style',
-          value: '<div style="margin-bottom: 40px">%TEMPLATE%</div>'
-        },
-        {
-          type: 'slot',
-          name: 'meta',
-          value: '<slot name="meta"></slot>'
-        },
-        {
-          type: 'attribute',
-          name: 'variant',
-          value: 'neutral-100'
-        }
-      ]
-    });
-  }
+export const HeaderAndMainSlot = {
+  render: () => html`
+    <sd-teaser>
+      <h3 slot="headline">Simple teaser</h3>
+      <p>
+        Quis ut ex cupidatat proident cillum ullamco ea aute ad laborum aliqua incididunt sint ipsum. Elit enim
+        reprehenderit aliquip officia in minim. Eu ipsum pariatur dolor. Do ex in cupidatat anim aliqua sint voluptate
+        sunt nulla incididunt. Cupidatat officia reprehenderit est cupidatat et id officia ut. Exercitation id pariatur
+        elit occaecat ad Lorem nisi sunt pariatur do aute aliqua magna irure incididunt.
+      </p>
+    </sd-teaser>
+  `
 };
 
 /**
```

Comment:
Pls move variant section further up as the 1st section to appear underneath the props.